test/e2e/network_test.go

```diff
@@ -443,8 +443,12 @@ func (tc *testContext) getPodIP(selector metav1.LabelSelector) (string, error) {
 // getWindowsServerContainerImage gets the appropriate WindowsServer image based on VXLAN port
 func (tc *testContext) getWindowsServerContainerImage() string {
 	var windowsServerImage string
-	if tc.hasCustomVXLAN {
-		// If we're using a custom VXLANPort we need to use 2004
+	if tc.CloudProvider.GetType() == config.VSpherePlatformType {
+		// If we're on vSphere we will use 2022
+		windowsServerImage = "mcr.microsoft.com/powershell:lts-nanoserver-ltsc2022"
+	} else if tc.CloudProvider.GetType() == config.NonePlatformType {
+		// TODO: On platform=none we have to use 2004 since the job points to the WS2004 golden image via release repo
```

Comment:
do we have a ticket to track this one? Don't want this to be left like this without any tracking.

Reply:
Yes, captured as part of https://issues.redhat.com/browse/WINC-755.